template/flake.nix

```diff
@@ -54,6 +59,7 @@
       channelsConfig = {
         # allowBroken = true;
         allowUnfree = true;
+        # allowUnsupportedSystem = true;
```

Comment:
Was this needed for M1?

Reply:
The unsupported system? I don't remember actually why I added this, maybe when I was testing M1 back then. Either way, it shouldn't matter since it's commented but I can remove it if you think it can confuse people.

Reply:
I don't think it matters that much.